platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -9,20 +9,26 @@ Staging reviews test the [happy path](https://en.wikipedia.org/wiki/Happy_path)
 ## Required accessibility checklist for staging review
 
 - [ ] [Axe scans in daily development](#axe-scans-in-daily-development)
-- [ ] [Axe scans in end-to-end tests](#axe-scans-in-end-to-end-tests)
-- [ ] [Color tests](#color-tests)
-- [ ] [Content resize check](#content-resize-check)
+- [ ] [Axe scans included in end-to-end tests](#axe-scans-in-end-to-end-tests)
+- [ ] [Color tests for proper contrast and colorblindness](#color-tests)
+- [ ] [Content zoomed to 200%, 300%, 400%](#content-resize-check)
 - [ ] [Keyboard navigation check](#keyboard-navigation-check)
-- [ ] [Screen reader tests](#screen-reader-tests)
+- [ ] [Screen reader tests (VoiceOver + Safari or NVDA + Firefox)](#screen-reader-tests)
+
+## Recommended accessibility checklist for staging review
+
+- [ ] [Keyboard end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/testing/508-automated-testing.md) - These tests are currently written in Nightwatch.js, and will be ported to Cypress.io in the future
+- [ ] [Axe-coconut scans in daily development](https://www.deque.com/blog/test-leading-edge-accessibility-axe-coconut-axe-core-3-0/)
+- [ ] [WAVE tool spot checks](https://wave.webaim.org/)
 
 
 ## How to document that your team has completed the checklist
 
 1. Choose one of these 2 options: 
 
-    * **(preferred)** Use the [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/projects/overview/13). The accessibility test cases template is available in TestRail for you to copy into your test plan and includes all the required accessibility checks.   \
+    * **(preferred)** Use the [TestRail VSP low-level accessibility testing checklist](https://dsvavsp.testrail.io/index.php?/projects/overview/13). The accessibility test cases template is available in TestRail for you to copy into your test plan and includes all the required accessibility checks.   \
 OR
-    *   Copy the checklist from this doc into a doc in your product folder. _Note that this artifact can be used for the Collaboration Cycle staging review._
+    *   Copy the required and recommended checklists from this doc into a doc in your product folder. _Note that this artifact can be used for the Collaboration Cycle staging review._
```

Comment:
Consider removing this option, and instead include the checklist of requirements in the staging review request — which may include more checklists for other practice areas. I'd like to minimize effort for VFS teams, as they're already having a tough time incorporating this into their workflows.

Reply:
In the below "How to request help" section, it asks teams to create 508 issue tickets for questions. I'm finding teams have questions as they're doing their "low-level" tests. Might we 1) supplement guidance provided to include info that if they see issues in the header, footer, or with the Feedback button to disregard; how to handle background color manual checks, etc, and 2) revise to say to open 508 issues for issues discovered, rather than to use for questions?

Reply:
For each of the "low-level" tests teams need guidance on what to disregard, as they're currently asking lots of questions and including screenshots of axe scan reports, etc.

Reply:
Below it says

> After you request a staging review, the accessibility specialist reviews your work and documents any defects in ZenHub tickets. Issues with a 508 defect rating of “0” or “1” are launch blockers. Others can be fixed post-launch.  

Might we be specific that 508-defect-2 and 508-defect-3 issues **must** be fixed within the next [insert number] sprints?

For 508-defect-4 issues, consider clarification these are enhancements, and decisions should be documented within [insert number] sprints. 

This may help resolve the backlog of accessibility issues.